MontyHallv2/Messages/ConstantDialogs.cs

```diff
@@ -0,0 +1,11 @@
+namespace MontyHallv2.Messages;
+
+public static class ConstantDialogs
+{
+    public const string InvalidUserDoorSelectionInput = "Please enter a number from 1 to 3";
+    public const string InvalidUserSwitchOrStay = "Please enter y or n";
+    public const string WinningCar = "Congratulations you have won the car";
+    public const string LosingCar = "Sorry you have not won the car :(";
+    public const string PlayerSelectADoor = "Please select a Door from 1 to 3";
+    public const string ToSwitchOrStay = "If you Would like to switch your door please enter 'y' or 'n' to stay";
```

Comment:
```suggestion
    public const string ToSwitchOrStay = "If you would like to switch your door please enter 'y', or 'n' to stay";
```
could be clearer as to mapping "y to switch" and "n to stay"